src/components/tracked-controls.js

```diff
@@ -145,12 +145,12 @@ module.exports.Component = registerComponent('tracked-controls', {
     // Compose pose from Gamepad.
     pose = controller.pose;
     // If no orientation, use camera.
-    if (pose.orientation) {
+    if (pose && pose.orientation) {
```

Comment:
from slack...

```example: it is perfectly legal for the user to have a rift and paired touches, but the touches are not currently powered up.  ideally that might appear as gamepad present with usual touch id, but connected false and (i would argue) undefined or null pose.

so that the app can show a message "please turn on your touch controllers"```